### PR TITLE
Flesh out gRPC guides, consolidate installation docs

### PR DESCRIPTION
**Pull request by MaxFangX**
Python and Javascript gRPC guides
- Now serve as complete standalone guides programmatically pulled into [dev.lightning.community](http://dev.lightning.community)
- Contains code examples for Simple, Response-streaming, and Bidirectional RPCs

Consolidate installation docs

Remove characters from the docker guide that break utf-8 encoding (you have to decode it with `latin-1` codec which is too hacky to implement as special logic in the dev site rendering code

### CODE REVIEW

File: docs/grpc/python.md [hunk 1]
```diff
@@ -37,29 +39,83 @@ $ source lnd/bin/activate
...
+#### Simple RPC
+
+```python
+# Retrieve and display the wallet balance
 response = stub.WalletBalance(ln.WalletBalanceRequest(witness_only=True))
-print(response.balance)
+print response.balance
 ```
 
+#### Response-streaming RPC
```
Comment (bryanvu) [inline docs/grpc/python.md]: I would mention the same prerequisites as for the JS versions.

**Comment by bryanvu** [comment]
Nice work Max! Just added a few minor comments. Let me know if you have any questions.

File: docs/INSTALL.md [hunk 2]
```diff
@@ -1,50 +1,106 @@
...
 
+    # On Mac OS X
+    brew install go
+    ```
+    More detailed installation instructions can be found
+    [here](http://golang.org/doc/install). 
+
+    At this point, you should set your `$GOPATH` environment variable, which
+    represents the path to your workspace. By default, `$GOPATH` is set to
+    `~/go`. Be sure to set your `$GOPATH` every time you open a new terminal
```
Comment (MaxFangX) [inline docs/INSTALL.md]: Will update in tutorial as well.

File: docs/INSTALL.md [hunk 3]
```diff
@@ -1,50 +1,106 @@
...
+    window.
+    ```
+    export GOPATH=~/projects/lightning
```
Comment (MaxFangX) [inline docs/INSTALL.md]: I figured that people may want to keep their Go workspaces separate. In any case, I implemented your above suggestion of recommending they put this step (as well as the `$GOPATH/bin` step below) into their .bashrc so I don't think it's an issue.

File: docs/grpc/javascript.md [hunk 4]
```diff
@@ -55,19 +54,96 @@ You should get something like this in your console:
...
+})
+.on('end', function() {
+  // The server has finished sending
+})
+.on('status', function(status) {
+  // Process status
+  console.log("Current status" + status);
+});
+```
+Now, create an invoice for your node at `localhost:10009` and send a payment to
```
Comment (MaxFangX) [inline docs/grpc/javascript.md]: Great suggestion - updated for Python guide as well.

**Comment by MaxFangX** [comment]
Changes implemented, rebased, and ready for merge

**Pull request merged**